File: bookhub_store_back/src/main/java/com/study/bookhub_store_back/dto/auth/response/LoginResponseDto.java
package com.study.bookhub_store_back.dto.auth.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.Getter;

@Getter
@Builder
@AllArgsConstructor
public class LoginResponseDto {
    private String token;
    private int exprTime;
    private CustomerResponseDto user;
}

File: bookhub_store_front/src/dtos/auth/LoginResponse.dto.ts
export interface LoginResponseDto {
  token: string;
  exprTime: number;
  user: CustomerResponseDto;
}

export interface CustomerResponseDto {
  id: number;
  email: string;
  name: string;
  nickname: string;
  phoneNumber: string;
  profileImageUrl: string;
  socialProvider: string;
}